openapi-generator-ignore

```diff
@@ -0,0 +1,5 @@
+**/src/main/typescript/generated/openapi/typescript-axios/git_push.sh
+**/src/main/typescript/generated/openapi/typescript-axios/.npmignore
+**/src/main/typescript/generated/openapi/typescript-axios/.gitignore
+**/src/main/kotlin/generated/openapi/kotlin-client/docs/**
+**/src/main/kotlin/generated/openapi/kotlin-client/gradle/**
```

Comment:
@jagpreetsinghsasan Question: Do we have to specify these paths here? E.g., couldn't it just work if we had the contents of this file defined with the same content as we used to have in the per-package ignore files (the ones that this one will now supersede)?

Reply:
@petermetz It works if we have this file content changed to
```
**/git_push.sh
**/.npmignore
**/.gitignore
**/generated/openapi/kotlin-client/docs/**
**/generated/openapi/kotlin-client/gradle/**
```
If we have just `.npmignore` then the generator will not try to generate the root `.npmignore` which is not what this is intended for (a tradeoff for using a global generator-ignore file)
So shall I change it to the above one, or do you have any suggestions?